pages/common/croc.md

```diff
@@ -0,0 +1,32 @@
+# croc
+
+> Send and recieve files easily and security over any network.
+> More information: <https://github.com/schollz/croc>.
+
+- Send a file or folder:
+
+`croc send {{path/to/file}}`
+
+- Send a file or folder with a specific passphrase:
+
+`croc send --code {{your-own-created-passphrase}} {{path/to/file}}`
+
+- Recieve a file or folder on reciving machine:
+
+`croc {{the-passphrase-to-recieve-a-file}}`
+
+- Send over a custom relay and custom relay:
+
+`croc --relay {{ip-to-relay}} send {{path/to/file}}`
+
+- Recieve over a custom relay and custom relay:
+
+`croc --relay {{ip-to-relay}} {{the-passphrase-to-recieve-a-file}}`
```

Comment:
These bracket names need to be changed.

Reply:
Changed to shorter and if multi word snake-case.